fuel-vm/src/interpreter/blockchain.rs

```diff
@@ -649,36 +629,40 @@ struct CodeCopyCtx<'vm, S, I> {
 impl<'vm, S, I> CodeCopyCtx<'vm, S, I> {
     pub(crate) fn code_copy(
         mut self,
-        a: Word,
-        b: Word,
-        c: Word,
-        d: Word,
+        dst_addr: Word,
+        contract_id_addr: Word,
+        contract_offset: Word,
+        length: Word,
```

Comment:
Do we need to worry about padding here? Or is it the responsibility of the `CCP` user?

Reply:
hmm it's not in the specs. I guess since this isn't meant to be executable code it's not so relevant here.